card.rb

```diff
@@ -21,4 +25,10 @@ def value
       @value = 11
     end
   end
+
+  private
+
+  def validate!
+    raise 'Неверный формат карты!' if !CARDS.include?(@name)
```

Comment:
лучше 2 валидации
1. масть входит в коллекцию мастей
2. значение входит в возможные значения

если очень хочется, то можно еще написать валидацию уникальности, т.е. что таких карт больше нет в колоде

Reply:
рекомендуют избегать (по возможности) сочетаний типа `if !`
можно так:
```suggestion
    raise 'Неверный формат карты!' unless CARDS.include?(@name)
```